mw-webapp/src/component/header/Header.module.scss

```diff
@@ -75,7 +78,7 @@ $userNameFontSize: 16px;
 }
 
 .socialMediaIcon {
-  border: 2px solid var(--secondaryStrokeColor);
+  border: $secondaryBorder solid var(--secondaryStrokeColor);
   border-radius: $tertiaryBorderRadius;
   background-color: var(--iconBgColor);
   fill: var(--quaternaryBgColor);
```

Comment:
fill is not bg

Reply:
"fill" is an attribute or property that defines the color used to paint the interior of a shape. you mean we need to create new token quanternaryIconColor like quanternaryBgColor?

Reply:
no, all okey, my mistake